### DISCUSSION TITLE
Plugged in while turned off

### DISCUSSION THREAD

**Discussion by KissDanielMark** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#119] [Q&A]
What will happen if I turn off the computer. Will it stay at the desired % or will it start charging to 100%?

**Comment by davidwernhart** [comment]
Hey @KissDanielMark 
Using the new M1 method, every modification should be reset after you quit AlDente (or restart).
If you are on an Intel MacBook und using the "Classic Key" Option, the maximum charging value will persist restarts and you closing the app. In this case, you can reset the value bei either setting 100% in AlDente or performing a SMC reset.

Hope that Helps!
David
